pom.xml

```diff
@@ -53,6 +53,7 @@
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.aws-sdk.version>1.12.85</dep.aws-sdk.version>
         <dep.okhttp.version>3.14.9</dep.okhttp.version>
+        <dep.joda.version>2.10.13</dep.joda.version>
```

Comment:
can you send PR to update joda in airbase too?

Reply:
@losipiuk  https://github.com/airlift/airbase/pull/297